x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/Authentication.java

```diff
@@ -130,6 +136,43 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeMap(metadata);
     }
 
+    /**
+     * Check whether this authentication object is owned by the same user as the given authentication object.
+     * The owners are considered to be the same if:
```

Comment:
```suggestion
Checks whether the user or API key of the passed in authentication can access the resources owned by the user or API key of this authentication.
```
The authentication being owned by the user is a bit confusing.